Plugins/GRpluginStands.txt

```diff
@@ -1919,87 +2361,160 @@ PRIORITY:+1
 ADEP:EN
 NOTADEP:ENSB,ENAS,ENSA
 USE:A
 CODE:C
 
 //STAND 3
-STAND:ENTO:3:N059.10.45.280:E010.15.14.370:18
+STAND:ENTO:3
+COORD:N059.10.45.805:E010.15.13.967
+COORD:N059.10.44.763:E010.15.14.007
+COORD:N059.10.44.787:E010.15.17.738
+COORD:N059.10.45.825:E010.15.17.707
+COORD:N059.10.45.805:E010.15.13.967
 ADEP:EN
 NOTADEP:ENSB,ENAS,ENSA
 USE:A
 CODE:C
 
 //STAND 4
-STAND:ENTO:4:N059.10.43.970:E010.15.14.390:18
+STAND:ENTO:4
+COORD:N059.10.44.492:E010.15.13.948
+COORD:N059.10.43.450:E010.15.13.988
+COORD:N059.10.43.474:E010.15.17.720
+COORD:N059.10.44.512:E010.15.17.689
+COORD:N059.10.44.492:E010.15.13.948
 ADEP:EN
 NOTADEP:ENSB,ENAS,ENSA
 USE:A
 CODE:C
 
 //STAND 5
-STAND:ENTO:5:N059.10.42.660:E010.15.14.390:18
+STAND:ENTO:5
+COORD:N059.10.43.190:E010.15.13.981
+COORD:N059.10.42.148:E010.15.14.021
+COORD:N059.10.42.172:E010.15.17.752
+COORD:N059.10.43.210:E010.15.17.722
+COORD:N059.10.43.190:E010.15.13.981
 ADEP:EN
 NOTADEP:ENSB,ENAS,ENSA
 USE:A
 CODE:C
 
 //STAND 10
-STAND:ENTO:10:N059.10.47.950:E010.15.11.770:18
+STAND:ENTO:10
+COORD:N059.10.47.645:E010.15.12.961
+COORD:N059.10.47.641:E010.15.10.919
+COORD:N059.10.49.540:E010.15.10.900
+COORD:N059.10.49.544:E010.15.12.967
+COORD:N059.10.47.645:E010.15.12.961
 CODE:C
 USE:A
 SCHENGEN
 
 //STAND 11
-STAND:ENTO:11:N059.10.47.950:E010.15.09.220:18
+STAND:ENTO:11
+COORD:N059.10.47.632:E010.15.10.301
+COORD:N059.10.47.629:E010.15.08.259
+COORD:N059.10.49.527:E010.15.08.240
+COORD:N059.10.49.531:E010.15.10.308
+COORD:N059.10.47.632:E010.15.10.301
 CODE:C
 USE:A
 SCHENGEN
 
 //STAND 12
-STAND:ENTO:12:N059.10.47.940:E010.15.06.670:18
+STAND:ENTO:12
+COORD:N059.10.47.619:E010.15.07.737
+COORD:N059.10.47.616:E010.15.05.696
+COORD:N059.10.49.514:E010.15.05.676
+COORD:N059.10.49.518:E010.15.07.744
+COORD:N059.10.47.619:E010.15.07.737
 CODE:C
 USE:A
 SCHENGEN
 
 //STAND 13
-STAND:ENTO:13:N059.10.47.930:E010.15.04.130:18
+STAND:ENTO:13
+COORD:N059.10.47.612:E010.15.05.103
+COORD:N059.10.47.609:E010.15.03.061
+COORD:N059.10.49.507:E010.15.03.042
+COORD:N059.10.49.511:E010.15.05.110
+COORD:N059.10.47.612:E010.15.05.103
 CODE:C
 USE:A
 NON-SCHENGEN
 
 //STAND 14
-STAND:ENTO:14:N059.10.48.030:E010.15.01.590:18
+STAND:ENTO:14
+COORD:N059.10.47.591:E010.15.02.558
+COORD:N059.10.47.588:E010.15.00.516
+COORD:N059.10.49.486:E010.15.00.497
+COORD:N059.10.49.490:E010.15.02.565
+COORD:N059.10.47.591:E010.15.02.558
 CODE:C
 USE:A
 NON-SCHENGEN
 
 //STAND 15
-STAND:ENTO:15:N059.10.48.020:E010.14.59.040:18
+STAND:ENTO:15
+COORD:N059.10.47.570:E010.14.59.908
+COORD:N059.10.47.567:E010.14.57.867
+COORD:N059.10.49.465:E010.14.57.847
+COORD:N059.10.49.469:E010.14.59.915
+COORD:N059.10.47.570:E010.14.59.908
 CODE:C
 USE:A
 NON-SCHENGEN
 PRIORITY:-1
 
 //STAND 32
-STAND:ENTO:32:N059.11.08.540:E010.15.22.460:12
+STAND:ENTO:32
+COORD:N059.11.08.925:E010.15.21.707
+COORD:N059.11.08.929:E010.15.23.221
+COORD:N059.11.08.147:E010.15.23.212
+COORD:N059.11.08.146:E010.15.21.720
+COORD:N059.11.08.925:E010.15.21.707
 CODE:B
 USE:BPH
```

Comment:
Probably not a good parking for business/corporate aircraft.. 
I believe they park outside the hangar for Access Oslo Executive Handling?

Reply:
You are right. Will change that.

Reply:
Fixed in https://github.com/Vatsim-Scandinavia/ENOR-Sectorfile/pull/406/commits/aaf1f35971767472e1e0cbfc9c7fc4e39319f9ff